docs/learn-staking.md

```diff
@@ -123,6 +123,14 @@ Based on the above rewards distribution, nominators in validator pool B get more
 than those in pool A because pool A has more overall stake. Sam has staked 50 DOT in pool A, but he
 only gets 8.3 in return, whereas Kitty gets 12.5 with the same amount of stake.
 
+There is an additional factor to consider in terms of rewards. While there is no limit to the number
+of nominators a validator may have, a validator does have a limit to how many nominators to which it
+can pay rewards. In Polkadot and Kusama, this limit is currently 64. A validator with more than 64
```

Comment:
I would additionally add that this can be changed via governance and may be a higher value in the future.

Reply:
good point

Reply:
It's specified in the runtime, so while technically the upgrade happens via on-chain governance, I think specifying that it happens via runtime upgrade makes more sense.